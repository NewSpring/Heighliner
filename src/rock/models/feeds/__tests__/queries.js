import Queries from "../queries";

describe("Feeds Queries", () => {
  it("has all needed queries", () => {
    expect(Queries).toMatchSnapshot();
  });
});
