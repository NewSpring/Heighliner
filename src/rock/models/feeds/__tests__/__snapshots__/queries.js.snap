// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feeds Queries has all needed queries 1`] = `
Array [
  "userFeed(
    filters: [String]!
    limit: Int = 20,
    skip: Int = 0,
    status: String = \\"open\\",
    cache: Boolean = true
  ): [Node]",
]
`;
